packages/app-form-builder/src/admin/components/FormEditor/Droppable.tsx

```diff
@@ -20,29 +20,34 @@ export interface DroppableCollectedProps {
     isOver: boolean;
 }
 
+export type DroppableUIElements = "row" | "field" | "conditionGroup" | "step";
```

Comment:
```suggestion
export type DraggableUIElement = "row" | "field" | "conditionGroup" | "step";
```

Notice the singular instead of plural.

P.S. At this point, `ui` name of the prop does sound ambiguous, but let's leave it as is for now (I'm aware if was like this before you started working on this).